src/components/v5/common/Modals/ManageMemberModal/consts.ts

```diff
@@ -1,32 +1,65 @@
+import { defineMessages } from 'react-intl';
+import { formatText } from '~utils/intl';
 import { SelectOption } from '~v5/common/Fields/Select/types';
 
+const actionMessages = defineMessages({
+  banMember: {
+    id: 'ManageMembers.Actions.Ban',
+    defaultMessage: 'Ban this member from chat access',
+  },
+  unbanMember: {
+    id: 'ManageMembers.Actions.Unban',
+    defaultMessage: 'Unban this member and allow chat access',
+  },
+  reduceReputation: {
+    id: 'ManageMembers.Actions.ReduceReputation',
+    defaultMessage: 'Reduce member’s reputation',
+  },
+  awardReputation: {
+    id: 'ManageMembers.Actions.AwardReputation',
+    defaultMessage: 'Award member with reputation',
+  },
+  addVerifiedMember: {
+    id: 'ManageMembers.Actions.AddVerified',
+    defaultMessage: 'Add member to the verified members',
+  },
+  removeVerifiedMember: {
+    id: 'ManageMembers.Actions.RemoveVerified',
+    defaultMessage: 'Remove from the verified members',
+  },
+  editPermissions: {
+    id: 'ManageMembers.Actions.EditPermissions',
+    defaultMessage: 'Edit this member’s permissions',
+  },
+});
+
 export const manageMemberActions: SelectOption[] = [
   {
-    label: 'Ban this member from chat access',
+    label: formatText(actionMessages.banMember) || '',
```

Comment:
If we're merging #1455 before this one, we have to remember to remove the fallbacks.